RELEASES.md

```diff
@@ -0,0 +1,44 @@
+# Release Process
+
+This section describes the release processes for tracking, preparing, and creating new L3AF releases. This includes information around the release cycles and guides for developers responsible for preparing upcoming stable releases.
+
+## Active development
+
+Active development is happening on the `main` branch, and a new version can be released from it in every 10-12 weeks.
+
+## Release Tracking
+
+Feature work for upcoming releases is tracked through GitHub Projects.
+
+## Release Versions
+
+We would be having a release version for every release. Minor releases are typically designated by incrementing 
+the Y in the version format X.Y.Z. New patch releases for an existing stable version X.Y.Z are published by incrementing
+the Z in the version format.
+
+## Stable releases
+
+Stable releases of L3AF include:
+
+* Extended maintenance window (any version released in the last 6/12 months).
```

Comment:
What is meant by "6/12"?   "6 to 12"?   "6"?   Undecided?
And what does "Extended maintenance window" really mean?  Just the bullets in 24-25?   If so maybe move this to a sentence above the bullet list instead of a bullet.